security/index.adoc

```diff
@@ -0,0 +1,85 @@
+[[security-intro]]
+= Introduction
+{product-author}
+{product-version]
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+:sourcedir: ../
+
+toc::[]
+
+[[security-intro-about-this-guide]]
+== About This Guide
+
+This guide provides a high-level walkthrough of the container security measures
+available in {product-title}, from the host layer to the container and
+orchestration layer to the user build and application layer. After reading this
+guide, you should be able to identify:
+
+- Why container security is important and how it compares with existing security standards.
+- Which container security measures are provided by the host (RHEL) layer and which are provided uniquely from {product-title}.
+- How to evaluate your container content and sources for vulnerabilities.
+- How to design your build and deployment process to proactively check container content.
+- How to control access to containers via authentication and authorization.
+- How networking and attached storage are secured in {product-title}.
+- Containerized solutions for API management and SSO.
+
+Basic concepts covered in this guide include links to further reading provided
+elsewhere in the {product-title} product documentation as well as other Red Hat
+and external documentation.
+
+[[security-intro-what-are-containers]]
+== What Are Containers?
+
+Containers have broad appeal for the ability they provide to easily package an
+application and all its dependencies into a single image that can be promoted
+from development, to test, to production, without change.
+
+Containers make it easy to ensure consistency across environments and multiple
+deployment targets: physical servers, virtual machines (VMs), and private or
+public cloud. All of this helps teams to develop and manage the applications
+that drive business value with greater agility.
+
+Therefore, depending on whether you are asking a system administrator or an
+application developer, the benefits of containers may sound slightly different:
+
+// image::whatarecontainers.png["What Are Containers?", align="center"]
+
+[options="header",frame="all",grid="none"]
+|===
+|INFRASTRUCTURE |APPLICATIONS
+
+|Sandboxed application processes on a shared Linux OS kernel
+|Package my application and all of its dependencies
+
+|Simpler, lighter, and denser than virtual machines
+|Deploy to any environment in seconds and enable CI/CD
+
+|Portable across different environments
+|Easily access and share containerized components
+|===
+
+[[security-intro-enterprise]]
+== Securing Containers for Enterprise Use
+
+However, enterprise use of containers requires strong security. In many ways,
```

Comment:
Is there a reason this is starting with "However,"? I can't really see what it's linking to. I think if it's a new section it's safe to just jump into the info.